src/logged-evaluation.nix

```diff
@@ -31,30 +31,30 @@ let
   # gc rooting the resulting store path from this build will retain
   # references to all the store paths needed, preventing the shell's
   # actual environment from being deleted.
-  wrapped-project = drv: services: derivation (
-    drv.drvAttrs // {
-      name = "lorri-wrapped-project-${drv.name}";
+  wrapped-project = shell: services: derivation (
```

Comment:
Nitpick: this can be an arbitrary `drv`, so naming it `drv` does make sense.

Reply:
While it can be an arbitrary `drv`, in the context in which this is used, we'll always have something representing a shell and something representing services, so I think those names are preferable because the communicate the intention better.